### ISSUE TITLE
Get last output

### ISSUE THREAD

**Issue by blainesch** [blainesch/pry.js#24]
In some repl's you can obtain the last output by using a magical underscore variable. The best way I could see this happening is wrapping the code in an anonymous self-executing function like:

~~~ js
// starting with
var i = 10;
++i;

// Turns into
var _ = (function() {
  var i = 10;
  ++i;
})();
~~~

However, now we lose getting the last output. So we'd need another input decorator that would add an explicit return value:

~~~ js
// starting with
var i = 10;
++i;

// Turns into
var i = 10;
return ++i;
~~~

This will add a few new abstractions to the code.

**Comment by blainesch** [comment]
The return statement could be tricky too. Imagine the following:
~~~ js
{
  name: function() {
    var i;
    ++;
  }
}
~~~

The return statement should be on the first line:
~~~ js
return {
  name: function() {
    var i;
    ++;
  }
}
~~~

**Comment by blainesch** [comment]
Fixed with b7bbdb8b0709837ba9567dee8cf1d3a55fa7429a

Much simpler than I expected since we are using eval.
